decidim-admin/config/locales/en.yml

```diff
@@ -375,8 +375,8 @@ en:
         form:
           domain_too_short: Domain too short
         update:
-          error: Failed to update domain whitelist
-          success: Domain whitelist updated successfully
+          error: Failed to update domain allowlist
```

Comment:
Could we do the `whitelist` -> `allowlist` change in a separate PR please?

Because
a) these are not the only places what we need to change
b) these changes do not match the title of this PR